pages/index.tsx

```diff
@@ -30,9 +32,13 @@ export default function Home({ claimData, loading }: HomeProps): ReactElement {
       </Head>
       <WorkInProgress />
       <Header />
-      <Main loading={loading} />
+      <Main
+        loading={loading}
+        statusContent={scenarioContent.statusContent}
+        detailsContent={scenarioContent.detailsContent}
```

Comment:
Note: I broke this out into two arguments, rather than one. I don't feel super strongly about this, so let me know if anyone would prefer that we pass one object instead.